### PR TITLE
[DO NOT MERGE]Lock 10 update

### PR DESCRIPTION
**Pull request by vmartynets**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/client-platforms/jquery/01-login.md [hunk 1]
```diff
@@ -72,10 +72,23 @@ ${browser}
...
 ```js
-lock.getClient().getDelegationToken({
+var auth0 = new Auth0({
+  domain: AUTH0_DOMAIN,
+  clientID: AUTH0_CLIENT_ID,
+});
+
+...
+
+auth0.getClient().getDelegationToken({
```
Comment (woloski) [inline articles/client-platforms/jquery/01-login.md]: remove `.getClient()`

File: articles/server-platforms/aspnet-core/01-login.md [hunk 2]
```diff
@@ -239,21 +239,23 @@ Here is an example:
 <div id="root" style="width: 320px; margin: 40px auto; padding: 10px; border-style: dashed; border-width: 1px;">
     embeded area
 </div>
-<script src="https://cdn.auth0.com/js/lock-9.1.min.js"></script>
+<script src="http://cdn.auth0.com/js/lock/10.0.0-rc.2/lock.min.js"></script>
```
Comment (woloski) [inline articles/server-platforms/aspnet-core/01-login.md]: why cant we use an env var here?

File: snippets/client-platforms/jquery/setup.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ```js
 var lock = null;
 $(document).ready(function() {
-   lock = new Auth0Lock('${account.clientId}', '${account.namespace}');
+   lock = new Auth0Lock('${account.clientId}', '${account.namespace}', {
+       auth: { 
+           params: { scope: 'openid' } //Details: https://auth0.com/docs/scopes
```
Comment (woloski) [inline snippets/client-platforms/jquery/setup.md]: by default we should use scope: 'openid email'

**Comment by vmartynets** [comment]
@woloski thanks, done.

**Pull request merged**